### PR TITLE
Bot rewrite from JavaScript to Python 3.10

### PR DESCRIPTION
**Pull request by Tukul3**
Fixed the rotation and instability inconsistencies and integrated slash commands. Will get more features down the line :)

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+load_dotenv()
+
+bot = lightbulb.BotApp(token = os.getenv('DISCORD_TOKEN'))
+
+# Current rotation is 28th of February 2022 (day of EoD release). Rotation index 1
+
+# Json file containing the instability data
+with open("data.json","r") as file:
+    x = json.load(file)
```
[the remark this answers is not in the document]
Comment (Tukul3) [reply bot.py]: Oversight back from when it was first created for testing purposes, Resolved

File: bot.py [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+        for j in channels:
+            if j.name == "instabilities":
+                await bot.rest.create_message(channel=j.id,content=embed)
+
+    
+@bot.command
+@lightbulb.command("help","Shows list of commands")
+@lightbulb.implements(lightbulb.SlashCommand)
+async def help(ctx):
+    await ctx.respond("```md\nDiscretize [dT] Bot - Help menu\nBot now includes integrated slash commands. To ease use, you can tab or click options\n\t - /today - Shows the instabilities for today\n\t - /tomorrow - Shows the instabilities for tomorrow\n\t - /in x - Shows the instabilities in x days\n\t - /filter <level> <with_without> <instability_1> <instability_2>\nIf channel #instabilities is created, the bot will auto broadcast new instabilities every day at 01:01```")
```
[the remark this answers is not in the document]
Comment (Tukul3) [reply bot.py]: Fixed

File: bot.py [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+
+@bot.command
+@lightbulb.command("today","Shows today instabilities")
+@lightbulb.implements(lightbulb.SlashCommand)
+async def today(ctx):
```
[the remark this answers is not in the document]
Comment (Tukul3) [reply bot.py:131]: Adjusted, if i read the issue properly?

**Comment by Tukul3** [comment]
Bot should be ready. Live test is needed to test daily broadcast

File: bot.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (gw2princeps) [reply bot.py]: Nope, not yet. https://github.com/discretize/discretize-discord-bot-instabilities/pull/10/files#diff-36f6200cd8353111badb874140eba3af6055f4cc5839fbdeb9ede992620828c1R90-R104 
this part is the same for the three methods (tomorrow, today and in), right?

**Comment by gw2princeps** [comment]
You can also adjust the Procfile to include the python command that you use to launch the bot

File: bot.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Tukul3) [reply bot.py:140]: Yes, they were. Should be much better now

**Pull request merged**